tests/unit/Entity/EntityTest.php

```diff
@@ -374,34 +371,34 @@ public function testGetModificationTimeStampAsISONull()
      */
     public function testUpdateTimeStamps()
     {
-        $this->assertNull($this->concreteEntity->getCreationTimeStamp());
-        $this->assertNull($this->concreteEntity->getModificationTimeStamp());
-        $this->concreteEntity->updateTimeStamps();
-        $this->assertInstanceOf('\DateTime', $this->concreteEntity->getCreationTimeStamp());
-        $this->assertInstanceOf('\DateTime', $this->concreteEntity->getModificationTimeStamp());
+        $this->assertNull($this->testEntity->getCreationTimeStamp());
+        $this->assertNull($this->testEntity->getModificationTimeStamp());
+        $this->testEntity->updateTimeStamps();
+        $this->assertInstanceOf('\DateTime', $this->testEntity->getCreationTimeStamp());
+        $this->assertInstanceOf('\DateTime', $this->testEntity->getModificationTimeStamp());
     }
 
-    /**
-     * Test that time stamps serialize to a specified time zone.
-     *
-     * @return void
-     */
-    public function testSerializeTimeStampsTimeZone()
-    {
-        $timeStamp = new \DateTime('now', new \DateTimeZone('UTC'));
-        $this->concreteEntity->setCreationTimeStamp($timeStamp);
-        $this->concreteEntity->setTimeZone('America/Chicago');
-        $timeStamp->setTimeZone(new \DateTimeZone('America/Chicago'));
-        $timeStampISO = $timeStamp->format(\DateTime::ISO8601);
-        $concreteEntityData = array(
-            'id' => null,
-            'creator' => $this->testCreator,
-            'creationTimeStamp' => $timeStampISO,
-            'modifier' => $this->testCreator,
-            'modificationTimeStamp' => $timeStampISO,
-            'name' => null,
-        );
-    }
+//    /**
```

Comment:
should we remove?

Reply:
Yes, I think we can remove it because doesn't have an assertion